playbook-website/app/javascript/components/LayoutRight/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import { Outlet } from "react-router-dom"
+import DarkModeToggle from "../DarkModeToggle"
+import HeaderMobile from "../../images/pb_generic_header.jpg"
+import HeaderDesktop from "../../images/pb_generic_header_desktop.jpg"
+
+import {
+  Layout,
+  FlexItem,
+  Flex,
+  Background,
+  Button,
+  Title,
+  Body,
+} from "playbook-ui"
+
+export default function LayoutRight({ isMobile, dark }) {
```

Comment:
Should this be specific to the content/page? LayoutRight sounds like a generic component that will load more of the site in. When this as is won't be loaded when we move on to "kit show" for example.  Should this be "kit index"?

Reply:
The kit show page will still load... inside of the LayoutRight view. 
It loads as an `<Outlet />` in react-router. 

I separated this out because my App.tsx was getting too large. 
Since the page always renders a LeftLayout, which includes the Sidebar that we have, I went ahead and called them LeftLayout and RightLayout.